sites/public/lib/filterUrlLink.tsx

```diff
@@ -0,0 +1,17 @@
+import { encodeToFrontendFilterString, ListingFilterState } from "@bloom-housing/ui-components"
+
+export function getFilterUrlLink(eligibilityRequirements) {
+  /** This is where any logic for the filtered results from the eligibility questionnaire should be located.*/
+
+  const SENIOR_AGE = 62
+
+  const params: ListingFilterState = {}
+
+  if (eligibilityRequirements.age < SENIOR_AGE) {
+    params.seniorHousing = false
+  } else {
+    params.seniorHousing = true
+  }
+
+  return `/listings/filtered?page=${1}${encodeToFrontendFilterString(params)}`
```

Comment:
I don't think you need this `page=` here, since the default on the `filtered.tsx` page is 1.

Reply:
updated.